src/reducer.test.ts

```diff
@@ -237,7 +238,7 @@ describe('API_DATA_SUCCESS with payload entity', () => {
                     endpointKey: 'postData',
                 }
             },
-            entities: { data: {} },
+            entities: { information: { id: 1, data: 'json' } },
```

Comment:
liever echte normalizr response gebruiken. Deze test heeft nu een datastructuur die nooit voorkomt, omdat het niet is hoe normalizr data format, dus beetje onbetrouwbare test.